pkg/config/config_darwin.go

```diff
@@ -4,9 +4,14 @@ import (
 	"os"
 )
 
+// podman remote clients on darwin cannot use unshare.isRootless() to determine the configuration file locations.
 func customConfigFile() (string, error) {
 	if path, found := os.LookupEnv("CONTAINERS_CONF"); found {
 		return path, nil
 	}
 	return rootlessConfigPath()
 }
+
+func ifRootlessConfigPath() (string, error) {
```

Comment:
Sorry, I don't understand what you mean, can you clarify?

Reply:
I hate the function, name, I thought there was a way we could just call rootlessConfigPath() and have it return "", if not rootless.  And this would solve the problem.

Reply:
`func rootessConfigPath` returns the path whether it is rootless or not, but this needs to be handled a specfic _darwin file since `unshare.IsRootless()` but this function doesn't work on mac. I can the function to just `readRootlessConfig() bool` and have it return true always on mac and on linux, wrap `unshare.IsRootless()` if that sounds better?